ricedb/ricemain.py

```diff
@@ -450,11 +450,13 @@ def create_rice(self, prog_name, for_upload=False):
             json.dump(json_data, fout, ensure_ascii=False)
         self.update_localdb(rice_name, prog_name)
         for rice_file in file_list:
-            if not os.path.exists(directory + rice_file['location'] + rice_file['filename']):
+            if not os.path.exists(directory + rice_file['location'] + rice_file['filename']) and not os.path.islink(directory + rice_file['location'] + rice_file['filename']):
                 self.renderer.alert("Could not find " +
                                     directory + rice_file['location'] + rice_file['filename'])
                 raise error.corruption_error("Could not find specified files")
-            os.rename(directory + rice_file['location'] + rice_file['filename'], "./" + rice_file['filename'])
+            if not os.path.exists(rice_file['location']):
```

Comment:
@Luminarys just for future reference, we're going to need to clean up ricemain at some point soon, it's doing a lot, as we discussed